setup.sh

```
@@ -0,0 +1,89 @@
+
+# Copy project to aivo user directory
+echo -e "${YELLOW}📂 Setting up project directory...${NC}"
+sudo mkdir -p /home/aivo/aivo-py
+sudo cp -r . /home/aivo/aivo-py/
+sudo chown -R aivo:aivo /home/aivo/aivo-py
+sudo chmod +x /home/aivo/aivo-py/app/entry.py
+sudo chmod +x /home/aivo/aivo-py/app/scheduler.py
+
+# Update service file paths
+echo -e "${YELLOW}🔧 Updating service files...${NC}"
+sed -i 's|/home/aivo/aivo-py|'$(pwd)'|g' aivo-recorder.service
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)